internal/statesync/reactor_test.go

```diff
@@ -154,23 +154,21 @@ func setup(
 
 	logger := log.NewNopLogger()
 
-	var err error
-	rts.reactor, err = NewReactor(
+	rts.reactor = NewReactor(
 		ctx,
 		factory.DefaultTestChainID,
 		1,
 		*cfg,
 		logger.With("component", "reactor"),
 		conn,
 		chCreator,
-		rts.peerUpdates,
+		func(_ context.Context) *p2p.PeerUpdates { return rts.peerUpdates },
```

Comment:
```suggestion
		func(context.Context) *p2p.PeerUpdates { return rts.peerUpdates },
```

It's fine with `_` too, but that's mainly useful in named functions where it shows up in godoc.